src/routers/customRouter.js

```diff
@@ -0,0 +1,69 @@
+import { Router } from "express";
+import { Jwt } from "jsonwebtoken";
+import { users } from "../data/mongo/manager.mongo.js";
+
+export default class CustomRouter {
+    constructor () {
+        this.Router = Router()
+        this.init()
+    }
+    getRouter() {
+        return this.Router;
+    }
+    init () {}
+    applyCbs (cbs) {
+        // cbs es un array de callbacks ( ej todos los middleware que necesita el endopint)
+        return cbs.map(each=>async(...params)=>{
+            try {
+                await each.apply(this,params)
```

Comment:
this,paramas? Con coma iría? O como acceso a un método del objeto this? Con un punto?